libraries/shared/factories/event/repository.go

```diff
@@ -24,7 +24,6 @@ import (
 type Repository interface {
 	Create(headerID int64, models []interface{}) error
 	MarkHeaderChecked(headerID int64) error
-	MissingHeaders(startingBlockNumber, endingBlockNumber int64) ([]core.Header, error)
```

Comment:
This transformer specific `MissingHeaders` method was never being used. Instead [this](https://github.com/vulcanize/vulcanizedb/blob/staging/libraries/shared/repository/repository.go#L47) generic `MissingHeaders` method is being called the [watcher](https://github.com/vulcanize/vulcanizedb/blob/staging/libraries/shared/watcher/event_watcher.go#L101) and then the headers are passed into `Transformer.Execute`.